zhaquirks/osram/switchmini.py

```diff
@@ -0,0 +1,129 @@
+"""Osram Smart+ Switch Mini device."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+    PollControl,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from . import OSRAM
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+    COMMAND,
+    COMMAND_ON,
+    MODELS_INFO,
+    BUTTON_1,
+    ENDPOINT_ID,
+    COMMAND_STEP_ON_OFF,
+    COMMAND_STOP,
+    BUTTON_2,
+    BUTTON_3,
+    LONG_RELEASE,
+    LONG_PRESS,
+    COMMAND_MOVE_TO_LEVEL_ON_OFF,
+    COMMAND_OFF,
+    COMMAND_MOVE,
+)
+
+OSRAM_DEVICE = 0x0810
```

Comment:
Remove this and replace it with https://github.com/zigpy/zigpy/blob/29e362563f31bc8e77462ac5b2fbd78c40aa926f/zigpy/profiles/zha.py#L64